jobs/basic-company/main.go

```diff
@@ -0,0 +1,178 @@
+// Copyright 2016 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"math/rand"
+	"time"
+
+	"golang.org/x/oauth2/google"
+	jobs "google.golang.org/api/jobs/v2"
+)
+
+const requestDeadline = 10 * time.Second
+
+func prettyFormat(v interface{}) string {
+	j, err := json.MarshalIndent(v, "", "    ")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	return string(j)
+}
+
+// [START basic_company]
```

Comment:
I tend to prefer to avoid words like simple, basic, etc. Is there another word this could use?